sdks/python/apache_beam/runners/interactive/recording_manager.py

```diff
@@ -454,8 +455,12 @@ def record(self, pcolls, max_n, max_duration):
           'options is deprecated since First stable release. References to '
           '<pipeline>.options will not be supported',
           category=DeprecationWarning)
-      pf.PipelineFragment(list(uncomputed_pcolls),
```

Comment:
Can you please add a unit test for asserting that the bug is fixed? It's good practice to add a unit test any time you fix a bug.

Reply:
That is a good idea, thanks! I've added a new unit test for this now just to check that the blocking is set to True when we use a GCS cache_root.